picture.go

```diff
@@ -471,39 +472,55 @@ func (f *File) GetPicture(sheet, cell string) (string, []byte, error) {
 
 // getPicture provides a function to get picture base name and raw content
 // embed in XLSX by given coordinates and drawing relationships.
-func (f *File) getPicture(row, col int, drawingXML, drawingRelationships string) (string, []byte, error) {
-	wsDr, _ := f.drawingParser(drawingXML)
-	for _, anchor := range wsDr.TwoCellAnchor {
+func (f *File) getPicture(row, col int, drawingXML, drawingRelationships string) (ret string, buf []byte, err error) {
+	var (
+		decoder             *xml.Decoder
+		wsDr                *xlsxWsDr
+		ok                  bool
+		anchor              *xdrCellAnchor
+		DecodeWsDr          *decodeWsDr
+		XlsxRelationship    *xlsxRelationship
+		DecodeTwoCellAnchor *decodeTwoCellAnchor
+	)
+
+	wsDr, _ = f.drawingParser(drawingXML)
+	for _, anchor = range wsDr.TwoCellAnchor {
 		if anchor.From != nil && anchor.Pic != nil {
 			if anchor.From.Col == col && anchor.From.Row == row {
-				xlsxRelationship := f.getDrawingRelationships(drawingRelationships,
+				XlsxRelationship = f.getDrawingRelationships(drawingRelationships,
```

Comment:
I propose to keep non-exportable variables, lowercase first letter.

Reply:
Lowercase variables match structure names.
This overrides the name of the structure inside the function and no longer makes it possible to use the structure. The local variable inside the function is not exportable.